lib/fog/compute/kubevirt/models/livevms.rb

```diff
@@ -5,15 +5,35 @@ module Fog
   module Compute
     class Kubevirt
       class Livevms < Fog::Collection
+        attr_reader :kind, :resourceVersion
+
         model Fog::Compute::Kubevirt::Livevm
 
         def all(filters = {})
-          load service.list_livevms(filters)
+          vms = service.list_livevms(filters)
+          kind = vms.kind
+          resourceVersion = vms.resourceVersion
+          load vms
         end
 
         def get(name)
           new service.get_livevm(name)
         end
+
+        def destroy(name)
+          begin
+            live_vm = get(name)
+          rescue KubeException
+            # the live virtual machine doesn't exist
+            live_vm = nil
+          end
+
+          # delete live vm
+          service.destroy_vm(name) unless live_vm.nil?
+
+          # delete offline vm
+          service.delete_offlinevm(name)
```

Comment:
not sure, is there any check that the vm exists at all? or is it ok to run delete_offlinevm on a name that is not there? or we want to raise an exception in that case?

Reply:
This would be fine. If call delete on the object which is not there we got to the state we wanted :)